addons/hr/static/src/core/common/@types/models.d.ts

```diff
@@ -1,5 +1,17 @@
 declare module "models" {
+    import { HrEmployee as HrEmployeeClass } from "@hr/core/common/hr_employee_model";
```

Comment:
Note: adding missing typing/models.

We'll need them in `hr_leave` overrides too anyway.